### DISCUSSION TITLE
release SDK 1.8.0

### DISCUSSION THREAD

**Discussion by jaggaer-fgeniola** [OP-TED/eForms-SDK#555] [Q&A] [answered]
Good morning,
I would like to know if the release of the SDK 1.8 for the end of June is confirmed.
If not, when would it be available?

Thank you.

**Comment by klakegg** [comment]
Any news on when we may expect version 1.8?

**Reply by AlexAS81** [reply]
We are also waiting impatiently.
When is it expected to be released?

**Comment by ferrakaop** [comment, marked as answer]
We are having some issues testing SDK 1.8 internally - we hope to have a clearer picture by the end of next week. Thanks for your patience.

You can already access the release candidate at https://github.com/OP-TED/eForms-SDK/tree/1.8.0-rc.3 to see what's changed since 1.7.0

**Comment by jaggaer-fgeniola** [comment]
Hi,
any news?
Thanks

**Comment by jaggaer-fgeniola** [comment]
Hi,
any news?
Thanks

**Comment by ferrakaop** [comment]
Thanks again for your patience. We are still having internal issues with SDK 1.8 and we're looking at ways to release it before the end of July.

**Comment by ferrakaop** [comment]
We plan to release SDK 1.8.0 this week, probably on Wednesday 26 July, and to activate in the Preview environment on Thursday.

If you have an account in Preview, you should have received an email about the service disruption on Wednesday morning when we deploy a new release of eNotices2.

**Reply by mdewinne** [reply]
Hi Karl, we assume that RC5 will constitute the final release of 1.8?

**Reply by ferrakaop** [reply]
Yes, the final release of 1.8.0 will have the same contents at RC5
